### PR TITLE
WIP: Fixes twitch panel

### PR DESCRIPTION
**Pull request by NejcZdovc**
Resolves https://github.com/brave/brave-browser/issues/3590
Resolves https://github.com/brave/brave-browser/issues/3417

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Windows
  - [x] macOS
  - [x] Linux
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on
  - [ ] Windows
  - [x] macOS
  - [ ] Linux
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone

## Test Plan:

- Enable Rewards
- Visit a twitch streamer (twitch.tv/jenoki works if you want a verified one)
- Even if user is streaming, play a VOD.
- After the video has been playing for a few seconds, open the panel.
- Make sure that you see publisher in the panel

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [x] Verify test plan is specified in PR before merging to source

### CODE REVIEW

File: vendor/bat-native-ledger/src/bat/ledger/internal/bat_get_media.cc [hunk 1]
```diff
@@ -874,6 +874,10 @@ void BatGetMedia::processTwitchMediaPanel(
     const ledger::VisitData& visit_data,
     const std::string& providerType,
     const std::string& publisher_blob) {
+  if (publisher_blob == "ignore") {
```
Comment (jasonrsadler) [inline vendor/bat-native-ledger/src/bat/ledger/internal/bat_get_media.cc]: Should we place this in `static_values.h` so we have one place for it?

Comment (NejcZdovc) [reply vendor/bat-native-ledger/src/bat/ledger/internal/bat_get_media.cc]: done, moved it into ledger, so we can use it in rewards code if needed

**Comment by NejcZdovc** [comment]
@jasonrsadler can you please re-test this one. If the problem is only for quick opens, let's merge this one and create new issue for it. Thank you

**Comment by NejcZdovc** [comment]
stale

**Pull request closed without merging**